traffic_ops/client/v13/server.go

```diff
@@ -131,3 +133,33 @@ func (to *Session) DeleteServerByID(id int) (tc.Alerts, ReqInf, error) {
 	err = json.NewDecoder(resp.Body).Decode(&alerts)
 	return alerts, reqInf, nil
 }
+
+func (to *Session) DeleteDeliveryServiceServer(dsID int, serverID int) (tc.Alerts, ReqInf, error) {
```

Comment:
This method conflicts with the method here: https://github.com/apache/trafficcontrol/blob/master/traffic_ops/client/v13/deliveryserviceserver.go#L59

Reply:
Fixed